treeherder/etl/taskcluster_pulse/handler.py

```diff
@@ -226,52 +224,61 @@ def buildMessage(pushInfo, task, runId, message):
 
 
 def handleTaskPending(pushInfo, task, message):
-    return buildMessage(pushInfo, task, message["runId"], message)
+    payload = message['payload']
+    return buildMessage(pushInfo, task, payload["runId"], payload)
 
 
 async def handleTaskRerun(pushInfo, task, message):
-    job = buildMessage(pushInfo, task, message["runId"]-1, message)
+    payload = message['payload']
```

Comment:
I was hitting an error here while doing my ingestion testing.  ``message`` in this case is already ``message['payload']`` from line 133 above.  So it should change in one of those places.  :)  Maybe here.  But it seems unclear for the param to be ``message`` when it is a field *within* a message.  So perhaps line 133 should pass in the whole message.

Reply:
Good find.  I missed changing the call to handleTaskRerun.